projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py

```diff
@@ -0,0 +1,634 @@
+def read_analog_input_regs_cn0414(
+        global_data: dict, address: list,
+        registers_number: int = 50, debug: bool = False) -> list:
+    """Read analog input registers.
+
+    Read analog input registers from CN0414 with function code 4.
+
+    Args:
+        instrument: Instrument object created by minimalmodbus
+        delay: Delay between MODBUS commands
+        address: MODBUS register start address
+        registers_number: Number of MODBUS register read (default value = 50)
+        debug: If True will print registers description and value
+            in a colored table (default value = False)
+
+    Returns:
+        Return a list of registers values
+
+    """
+    register_address = utilities.generate_register_address(address)
+    sleep(global_data["DELAY"])
+    registers = global_data["INSTRUMENT"].read_registers(
+        register_address, registers_number, functioncode=4)
+
+    if debug:
+        print(Fore.LIGHTMAGENTA_EX + '\nAnalog input registers at',
+              Fore.YELLOW + 'MODBUS address ' + str(address[1]))
+        labels = [
+            Fore.YELLOW + 'Channel 1 MSW', Fore.YELLOW + 'Channel 1 LSW',
+            Fore.YELLOW + 'Channel 2 MSW', Fore.YELLOW + 'Channel 2 LSW',
+            Fore.YELLOW + 'Channel 3 MSW', Fore.YELLOW + 'Channel 3 LSW',
+            Fore.YELLOW + 'Channel 4 MSW', Fore.YELLOW + 'Channel 4 LSW',
+            Fore.YELLOW + 'Channel 5 MSW', Fore.YELLOW + 'Channel 5 LSW',
+            Fore.YELLOW + 'Channel 6 MSW', Fore.YELLOW + 'Channel 6 LSW',
+            Fore.YELLOW + 'Channel 7 MSW', Fore.YELLOW + 'Channel 7 LSW',
+            Fore.YELLOW + 'Channel 8 MSW', Fore.YELLOW + 'Channel 8 LSW',
+            Fore.YELLOW + 'Channel 1 OWD', Fore.YELLOW + 'Channel 2 OWD',
+            Fore.YELLOW + 'Channel 3 OWD', Fore.YELLOW + 'Channel 4 OWD',
+            Fore.YELLOW + 'HART IN 01', Fore.YELLOW + 'HART IN 02',
+            Fore.YELLOW + 'HART IN 03', Fore.YELLOW + 'HART IN 04',
+            Fore.YELLOW + 'HART IN 05', Fore.YELLOW + 'HART IN 06',
+            Fore.YELLOW + 'HART IN 07', Fore.YELLOW + 'HART IN 08',
+            Fore.YELLOW + 'HART IN 09', Fore.YELLOW + 'HART IN 10',
+            Fore.YELLOW + 'HART IN 11', Fore.YELLOW + 'HART IN 12',
+            Fore.YELLOW + 'HART IN 13', Fore.YELLOW + 'HART IN 14',
+            Fore.YELLOW + 'HART IN 15', Fore.YELLOW + 'HART IN 16',
+            Fore.YELLOW + 'HART IN 17', Fore.YELLOW + 'HART IN 18',
+            Fore.YELLOW + 'HART IN 19', Fore.YELLOW + 'HART IN 20',
+            Fore.YELLOW + 'HART IN 21', Fore.YELLOW + 'HART IN 22',
+            Fore.YELLOW + 'HART IN 23', Fore.YELLOW + 'HART IN 24',
+            Fore.YELLOW + 'HART IN 25', Fore.YELLOW + 'HART IN 26',
+            Fore.YELLOW + 'HART IN 27', Fore.YELLOW + 'HART IN 28',
+            Fore.YELLOW + 'HART IN 29', Fore.YELLOW + 'HART IN 30']
+        utilities.print_table(
+            register_address, registers_number, labels, registers)
+    return registers
+
+
```

Comment:
nitpick: extra line

Reply:
Two lines are necessary to respect python linting.